### PR TITLE
common: fix KW/Cov issues

### PR DESCRIPTION
**Pull request by wlemkows**
This change is [Reviewable](https://reviewable.io/reviews/pmem/miniasync/52)

### CODE REVIEW

File: src/vdm.c [hunk 1]
```diff
@@ -54,6 +54,7 @@ vdm_memcpy(struct vdm *vdm, void *dest, void *src, size_t n, uint64_t flags)
 
 	struct vdm_operation_future future = {0};
 	future.data.op = vdm->op_new(vdm, &op);
```
Comment (pbalcer) [inline src/vdm.c:56]: add an `ASSERTne(future.data.op, NULL);` for now. We need proper error handling...

File: tests/memcpy_threads/memcpy_threads.c [hunk 2]
```diff
@@ -7,6 +7,7 @@
 #include "libminiasync.h"
 #include "core/os.h"
 #include "libminiasync/data_mover_threads.h"
+#include "../test_helpers.h"
```
Comment (pbalcer) [inline tests/memcpy_threads/memcpy_threads.c:10]: maybe just add `tests` to include directories.

**Review by wlemkows — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/miniasync/52)* status: 0 of 10 files reviewed, 2 unresolved discussions (waiting on @pbalcer)

___
*[src/vdm.c, line 56 at r1](https://reviewable.io/reviews/pmem/miniasync/52#-MwVP0YRNpwy8NIwodPI-r1-56:-MwV_UzrDdrzIrnoKGlP:b-896fix) ([raw file](https://github.com/pmem/miniasync/blob/20c29f3fedff550858a10bff73596a2bf5722f07/src/vdm.c#L56)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

add an `ASSERTne(future.data.op, NULL);` for now. We need proper error handling...
</blockquote></details>

Done.
___
*[tests/memcpy_threads/memcpy_threads.c, line 10 at r1](https://reviewable.io/reviews/pmem/miniasync/52#-MwVP0YRNpwy8NIwodPJ-r1-10:-MwVkkFw8QJtbk8sxs1q:b-896fix) ([raw file](https://github.com/pmem/miniasync/blob/20c29f3fedff550858a10bff73596a2bf5722f07/tests/memcpy_threads/memcpy_threads.c#L10)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

maybe just add `tests` to include directories.
</blockquote></details>

Done.

**Review by pbalcer — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/miniasync/52#-:-MwVmd9V-WKyJmQIZ1k6:bnfp4nl)* status: 0 of 10 files reviewed, all discussions resolved (waiting on @pbalcer)

**Pull request merged**